src/dydx-client.ts

```diff
@@ -13,22 +13,26 @@ import {
 
 export interface ClientOptions {
   apiTimeout?: number;
-  apiPrivateKey?: string | KeyPair;
+  apiKey?: string;
   ethSendOptions?: EthereumSendOptions;
   networkId?: number;
   starkPrivateKey?: string | KeyPair;
   web3?: Web3;
   web3Provider?: string | Provider;
+  secret?: string;
+  passphrase?: string;
 }
 
 export class DydxClient {
   readonly host: string;
   readonly apiTimeout?: number;
-  readonly apiPrivateKey?: string | KeyPair;
+  readonly apiKey?: string;
   readonly ethSendOptions?: EthereumSendOptions;
   readonly networkId: number;
   readonly starkPrivateKey?: string | KeyPair;
   readonly web3?: Web3;
+  readonly secret?: string;
+  readonly passphrase?: string;
```

Comment:
Our client options are fairly complex, I think it's worth organizing these as a group:

```
export interface ApiKeyCredentials {
  key: string,
  secret: string,
  passphrase: string,
}
...
apiKey?: ApiKeyCredentials
```